src/octoprint/util/comm.py

```diff
@@ -5371,6 +5395,23 @@ def _gcode_G4_sent(
         self._timeout = self._get_new_communication_timeout() + _timeout
         self._dwelling_until = time.monotonic() + _timeout
 
+    def _gcode_M106_sent(
+        self, cmd, cmd_type=None, gcode=None, subcode=None, *args, **kwargs
+    ):
+        s_match = regexes_parameters["intS"].search(cmd)
+
+        _fanspeed = 0
```

Comment:
Just `fanspeed` please, it's not a protected instance field but just a function local variable.